test/e2e/channel_dls_test.go

```diff
@@ -0,0 +1,35 @@
+//+build e2e
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"knative.dev/eventing/test/e2e/helpers"
+)
+
+// TestChannelDeadLetterSink tests DeadLetterSink
+func TestChannelDeadLetterSink(t *testing.T) {
```

Comment:
Why two and why both sub v1beta1?

Reply:
it's aligned w/ what evening-core does :shrug:

Reply:
So eventing-core is running the same exact test two times just with a different name? ahahha

Reply:
I fixed both :-) here and https://github.com/knative/eventing/pull/4061

stupid copy-paste error :rofl: